### PR TITLE
docs(list): add a11y list docs

### PR DESCRIPTION
**Pull request by aledavila**
Reference #491 

Adds a11y docs to list component.

Check to see content matches: https://pages.github.ibm.com/IBMa/Carbon_Integration/Doc/List-doc.html

### CODE REVIEW

**Comment by snidersd** [comment]
Document content LGTM. Thx

File: src/pages/components/list/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+        </StructuredListRow>
+        <StructuredListRow>
+          <StructuredListCell>
+            - macOS Mojave version 10.14.6 with VoiceOver
+            <br />
+            - Safari version 13.0.2
+            <br />- Carbon React version 7.7.1
+          </StructuredListCell>
+          <StructuredListCell>
+            <strong>VoiceOver(VO) Test:</strong>
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            <strong>VoiceOver(VO) test:</strong>
```

File: src/pages/components/list/accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+            <br />
+            - Chrome version 77.0.3865.90
+            <br />- Carbon React version 7.7.1
+          </StructuredListCell>
+          <StructuredListCell>
+            <strong>VoiceOver(VO) Test:</strong>
+            <OrderedList>
+              <ListItem>
+                Control-Option-Right Arrow to read the first list item. VO
+                announces, that there are 3 items in the list, then reads the
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
                announces that there are three items in the list, then reads the
```

File: src/pages/components/list/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+                item number, position and text.
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
                item number, position, and text.
```

File: src/pages/components/list/accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+              </ListItem>
+              <ListItem>
+                Control-Option-Right Arrow to read the next item and VO
+                announces you are in another list that contains 2 items.
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
                announces you are in another list that contains two items.
```

File: src/pages/components/list/accessibility.mdx [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+            <br />
+            - Safari version 13.0.2
+            <br />- Carbon React version 7.7.1
+          </StructuredListCell>
+          <StructuredListCell>
+            <strong>VoiceOver(VO) Test:</strong>
+            <OrderedList>
+              <ListItem>
+                Control-Option-Right Arrow to read the first list item. VO
+                announces, that there are 3 items in the list, then reads the
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
                announces that there are three items in the list, then reads the
```

File: src/pages/components/list/accessibility.mdx [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+                item number, position and text. (Note: Safari displays the first
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
                item number, position, and text. (Note: Safari displays the first
```

File: src/pages/components/list/accessibility.mdx [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+                list item as 0)
+              </ListItem>
+              <ListItem>
+                Control-Option-Right Arrow to read the next item and VO
+                announces you are in another list that contains 2 items.
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
                announces you are in another list that contains two items.
```

File: src/pages/components/list/accessibility.mdx [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+          <StructuredListCell head>Environment</StructuredListCell>
+          <StructuredListCell head>Results</StructuredListCell>
+        </StructuredListRow>
+      </StructuredListHead>
+      <StructuredListBody>
+        <StructuredListRow>
+          <StructuredListCell>
+            - Microsoft Windows 10
+            <br />
+            - Firefox Version 67
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            - Firefox version 67
```

File: src/pages/components/list/accessibility.mdx [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+            <br />
+            - JAWS 19.1810.64
+            <br />- Carbon React Version 7.7.1
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            <br />- Carbon React version 7.7.1
```

File: src/pages/components/list/accessibility.mdx [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+          <StructuredListCell head>Environment </StructuredListCell>
+          <StructuredListCell head>Results</StructuredListCell>
+        </StructuredListRow>
+      </StructuredListHead>
+      <StructuredListBody>
+        <StructuredListRow>
+          <StructuredListCell>
+            - Android Version 9 with Talkback
+            <br />
+            - Chrome Version 71.0.3578.98 (Official Build) (64-bit)
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            - Chrome version 71.0.3578.98 (Official Build) (64-bit)
```

File: src/pages/components/list/accessibility.mdx [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+            <br />- Carbon React Version 7.7.1
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            <br />- Carbon React version 7.7.1
```

File: src/pages/components/list/accessibility.mdx [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+          </StructuredListCell>
+          <StructuredListCell>
+            <strong>Talkback Test:</strong>
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            <strong>Talkback test:</strong>
```

File: src/pages/components/list/accessibility.mdx [hunk 13]
```diff
@@ -0,0 +1,353 @@
...
+        <StructuredListCell head>Environment </StructuredListCell>
+        <StructuredListCell head>Results</StructuredListCell>
+      </StructuredListRow>
+      <StructuredListBody>
+        <StructuredListRow>
+          <StructuredListCell>
+            - iOS Version 13.1.3 with VoiceOver
+            <br />
+            - Safari Version 13.1.3
+            <br />- Carbon React Version 7.7.1
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            <br />- Carbon React version 7.7.1
```

File: src/pages/components/list/accessibility.mdx [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+  <Column noGutterSm>
+    <StructuredListWrapper>
+      <StructuredListRow head>
+        <StructuredListCell head>Environment </StructuredListCell>
+        <StructuredListCell head>Results</StructuredListCell>
+      </StructuredListRow>
+      <StructuredListBody>
+        <StructuredListRow>
+          <StructuredListCell>
+            - iOS Version 13.1.3 with VoiceOver
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            - iOS version 13.1.3 with VoiceOver
```

File: src/pages/components/list/accessibility.mdx [hunk 15]
```diff
@@ -0,0 +1,353 @@
...
+            <br />
+            - Safari Version 13.1.3
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            - Safari version 13.1.3
```

File: src/pages/components/list/accessibility.mdx [hunk 16]
```diff
@@ -0,0 +1,353 @@
...
+            <br />- Carbon React Version 7.7.1
+          </StructuredListCell>
+          <StructuredListCell>
+            <strong>VoiceOver Test:</strong>
```
Comment (connor-leech) [inline src/pages/components/list/accessibility.mdx]:
```suggestion
            <strong>VoiceOver test:</strong>
```

**Pull request merged**